src/VisualStudio/Core/Def/Implementation/Venus/ContainedLanguage.cs

```diff
@@ -90,7 +90,7 @@ internal partial class ContainedLanguage<TPackage, TLanguageService>
                    languageService,
                    vbHelperFormattingRule: null)
         {
-            Contract.ThrowIfFalse(vbHelperFormattingRule != null);
+            Contract.ThrowIfTrue(vbHelperFormattingRule != null);
```

Comment:
The constructor says this is also being used by Live Share...will this break them?

Reply:
And if this is dead, should we just delete the assert entirely?

Reply:
> The constructor says this is also being used by Live Share...will this break them?

They are broken today.

> And if this is dead, should we just delete the assert entirely?

That is definitely the goal but I added the assertion under the assumption that this may never be fixed.

Reply:
I'll leave this since we're planning (🤞) to removing this code and this was really just an incorrectly written argument validation from the original change.